website/src/docs/docs.json

```diff
@@ -4,37 +4,110 @@
     "items": [
-      { "path": "index", "title": "Introduction" },
-      { "path": "getting-started", "title": "Getting Started" },
+      {
+        "path": "index",
+        "title": "Introduction"
+      },
+      {
+        "path": "getting-started",
+        "title": "Getting Started"
+      },
       {
         "path": "api-reference",
         "title": "API Reference",
-        "items": [{ "path": "index", "title": "Whatever" }]
+        "items": [
+          {
+            "path": "index",
+            "title": "Whatever"
+          }
+        ]
+      },
+      {
+        "path": "filtering",
+        "title": "Filtering",
+        "items": [
+          {
+            "path": "getting-started",
+            "title": "Getting Started"
+          },
+          {
+            "path": "filter-and-operations",
+            "title": "Filters & Operations"
+          },
+          {
+            "path": "string-filter",
+            "title": "- String Filter"
+          },
+          {
+            "path": "bool-filter",
+            "title": "- Boolean Filter"
+          },
+          {
+            "path": "comparable-filter",
+            "title": "- Comparable Filter"
+          },
+          {
+            "path": "object-filter",
+            "title": "- Object Filter"
+          },
+          {
+            "path": "list-filter",
+            "title": "- List Filter"
+          },
+          {
+            "path": "filter-customization",
+            "title": "Customization"
+          },
+          {
+            "path": "filter-naming-convention",
+            "title": "Naming Convention"
+          }
+        ]
       }
     ]
   },
   {
     "path": "bananacakepop",
     "title": "Banana Cake Pop",
     "description": "Use this GUI to get deep insights from any GraphQL endpoint.",
-    "items": [{ "path": "index", "title": "Introduction" }]
+    "items": [
+      {
+        "path": "index",
+        "title": "Introduction"
+      }
+    ]
   },
   {
     "path": "strawberryshake",
     "title": "Strawberry Shake",
     "description": "Configure a GraphQL client in .Net to fetch data from multiple GraphQL endpoints.",
-    "items": [{ "path": "index", "title": "Introduction" }]
+    "items": [
+      {
+        "path": "index",
+        "title": "Introduction"
+      }
+    ]
   },
   {
     "path": "marshmellowpie",
     "title": "Marshmellow Pie",
     "description": "Configure a GraphQL registry to keep track on any GraphQL server and client.",
-    "items": [{ "path": "index", "title": "Introduction" }]
+    "items": [
+      {
+        "path": "index",
+        "title": "Introduction"
+      }
+    ]
   },
   {
     "path": "greendonut",
     "title": "Green Donut",
     "description": "Configure data loaders and caching for GraphQL endpoints.",
-    "items": [{ "path": "index", "title": "Introduction" }]
+    "items": [
+      {
+        "path": "index",
+        "title": "Introduction"
+      }
+    ]
   }
-]
+]
```

Comment:
Please install `prettier` and let it decide how to format things.

Reply:
interesting 
![image](https://user-images.githubusercontent.com/14233220/80277296-4bc8a300-86ee-11ea-8773-afd92f61cb6c.png)

Reply:
Is it prettified? Sometimes it's not even if you see this flag :-D

Reply:
I will a git pre-hook for commits which will prettify everything to ensure it's pretty :-D